worldedit-bukkit/adapters/adapter-1_20_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_20_R3/PaperweightBlockMaterial.java

```diff
@@ -75,7 +66,7 @@ public boolean isFullCube() {
 
     @Override
     public boolean isOpaque() {
-        return blockState.isOpaque();
+        return blockState.canOcclude();
```

Comment:
`!canOcclude()`?

Reply:
I've orientated myself on the 1.20.3 source if the BlockState#isOpaque method there 
![image](https://github.com/IntellectualSites/FastAsyncWorldEdit/assets/27054324/852a5623-4cbc-4476-b3e7-b53c1e423d55)

Reply:
Yeah, the same was present in `1.20.4-R0.1-20240106.182028-62` - which is kinda weird as therefor isSolid == isTranslucent == isOpaque lol

Reply:
yeah, I'm dumb - didn't invert in isTranslucent